scipy/optimize/tests/test_linprog.py

```diff
@@ -23,8 +23,7 @@
 
 has_cholmod = True
 try:
-    import sksparse
-    from sksparse.cholmod import cholesky as cholmod
+    pass
 except ImportError:
     has_cholmod = False
```

Comment:
Not sure

Reply:
Yeah I'd just `noqa` all of the imports in `linprog` files.